data/human/intro missions.txt

```diff
@@ -29,15 +29,31 @@ mission "Intro [0]"
 			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
 		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
 		conversation
+			branch lampyrid
+				has "flagship model: Lampyrid Mk. II"
```

Comment:
```suggestion
				not "flagship model: Lampyrid Mk. II"
```

Reply:
The main conversation has been flipped now, so changing the condition is no longer necessary.